src/app/public/modules/sort/sort.component.html

```diff
@@ -5,7 +5,13 @@
     [messageStream]="dropdownController"
     [title]="'skyux_sort_button_label' | skyLibResources">
     <sky-dropdown-button>
-      <sky-icon icon="sort"></sky-icon>
+      <sky-icon *skyThemeIf="'default'"
```

Comment:
Should we add `<sky-docs-demo-control-panel-theme>` to the sort component's visual test?